app-frontend/src/app/core/services/histogram.service.js

```diff
@@ -0,0 +1,20 @@
+export default (app) => {
+    class HistogramService {
+        constructor($http, authService) {
+            'ngInject';
+            this.authService = authService;
+            this.$http = $http;
+        }
+
+        getHistogram(project, scenes) {
+            return this.$http.post(
+                `/tiles/${project}/histogram/?token=${this.authService.token()}`,
```

Comment:
ah right. I edited the webpack config to route /tiles requests to the tile server.
I'll change it to use the app_config url